app/models/sac_cas/future_role.rb

```diff
@@ -68,7 +68,9 @@ def validate_beitragskategorie?
     becomes_mitglied_role?
   end
 
+  # attributes to copy to new role
   def relevant_attrs
-    super.merge(person: person)
+    # remove attributes that don't exist in the database
+    super.merge(person: person).except("membership_years")
```

Comment:
I guess this needs some explaining. Probably fixes a bug not directly related to the wizard?

Reply:
the model contains attributes that are not in the database, but are shown in the table. those are calculated. It tried to persist those attributes and failed because of that